beacon-chain/rpc/eth/helpers/sync.go

```diff
@@ -81,6 +81,7 @@ type SyncDetailsJson struct {
 	SyncDistance string `json:"sync_distance"`
 	IsSyncing    bool   `json:"is_syncing"`
 	IsOptimistic bool   `json:"is_optimistic"`
+	ElOffline    bool   `json:"el_offline"`
```

Comment:
ugh i wish there was more pushback on the beacon API on this naming, why isn't this IsElOffline ...